kolibri/plugins/facility/assets/src/composables/useUserManagement.js

```diff
@@ -108,6 +113,15 @@ export default function useUserManagement({
     { immediate: true },
   );
 
+  watch([numAppliedFilters, search], (newValues, oldValues) => {
+    const [newNumFilters, newSearch] = newValues;
+    const [oldNumFilters, oldSearch] = oldValues;
+
+    if (newNumFilters !== oldNumFilters || newSearch !== oldSearch) {
```

Comment:
I like this change - it's more clear what is happening this way than before where it indexed into the arrays.